WorkflowSwiftUIExperimental/Sources/SwiftUIScreen.swift

```diff
@@ -77,17 +95,104 @@ private struct EnvironmentInjectingView<Content: View>: View {
 private final class ModeledHostingController<Model, Content: View>: UIHostingController<Content> {
     let modelSink: Sink<Model>
     let viewEnvironmentSink: Sink<ViewEnvironment>
+    var swiftUIScreenSizingOptions: SwiftUIScreenSizingOptions {
+        didSet {
+            updateSizingOptionsIfNeeded()
 
-    init(modelSink: Sink<Model>, viewEnvironmentSink: Sink<ViewEnvironment>, rootView: Content) {
+            if !hasLaidOutOnce {
+                setNeedsLayoutBeforeFirstLayoutIfNeeded()
+            }
+        }
+    }
+
+    private var hasLaidOutOnce = false
+
+    init(
+        modelSink: Sink<Model>,
+        viewEnvironmentSink: Sink<ViewEnvironment>,
+        rootView: Content,
+        swiftUIScreenSizingOptions: SwiftUIScreenSizingOptions
+    ) {
         self.modelSink = modelSink
         self.viewEnvironmentSink = viewEnvironmentSink
+        self.swiftUIScreenSizingOptions = swiftUIScreenSizingOptions
 
         super.init(rootView: rootView)
+
+        updateSizingOptionsIfNeeded()
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("not implemented")
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .clear
```

Comment:
Just curious: is this new and necessary for the PCS behavior we want?

Reply:
Oh yeah, whoops, meant to call this out or pull it into a separate PR...

It's not necessary to achieve the goal of this PR, but IMO it's generally expected, and annoying to work around if you want to have a transparent background when working with `Screens` like this. We also set this in `WorkflowHostingController`.

Epoxy sets this on [their hosting controller](https://github.com/airbnb/epoxy-ios/blob/e25f2915a004f64c27ebff7ed9ad41ec594a4cef/Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingController.swift#L38-L42) and they included a comment describing their motivation:
```swift
    // A `UIHostingController` has a system background color by default as it's typically used in
    // full-screen use cases. Since we're using this view controller to place SwiftUI views within
    // other view controllers we default the background color to clear so we can see the views
    // below, e.g. to draw highlight states in a `CollectionView`.
```

Happy to move this to a separate PR if we'd prefer!

Reply:
Thanks for explaining. Makes sense to me, and not important to break out IMO 👍

Reply:
worth adding a similar comment here?

Reply:
> worth adding a similar comment here?

Eh, maybe? I think it's probably pretty self-explanatory.
Happy to add one if anyone feels strongly though!

Reply:
> Maybe it's worth cribbing that comment into our implementation too.

2 comments about it seems like a signal that I should add the comment—I'll do so in a follow up!

Reply:
#278